includes/class-wporg-gp-translation-events-stats-calculator.php

```diff
@@ -51,33 +53,40 @@ public function totals(): WPORG_GP_Translation_Events_Stats_Row {
 }
 
 class WPORG_GP_Translation_Events_Stats_Calculator {
-	private string $ACTIONS_TABLE_NAME = WPORG_GP_Translation_Events_Translation_Listener::ACTIONS_TABLE_NAME;
-
 	/**
-	 * @throws Exception
+	 * Get stats for an event.
+	 *
+	 * @throws Exception When stats calculation failed.
 	 */
-	function for_event( WP_Post $event ): WPORG_GP_Translation_Events_Event_Stats {
-		$stats = new WPORG_GP_Translation_Events_Event_Stats;
+	public function for_event( WP_Post $event ): WPORG_GP_Translation_Events_Event_Stats {
+		$stats = new WPORG_GP_Translation_Events_Event_Stats();
 		global $wpdb;
 
-		$query = $wpdb->prepare( "
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.DirectQuery
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.NoCaching
+		// phpcs thinks we're doing a schema change but we aren't.
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.SchemaChange
+		$rows = $wpdb->get_results(
+			$wpdb->prepare(
+				"
 				select locale,
 					   sum(action = 'create') as created,
 					   count(*) as total,
 					   count(distinct user_id) as users
-				from $this->ACTIONS_TABLE_NAME
+				from wp_wporg_gp_translation_events_actions
```

Comment:
Maybe in the next iteration we should add the database prefix to the table, avoiding to hardcode its name.

```suggestion
				from {$wpdb->prefix}_gp_translation_events_actions
```